src/test/tests/hybrid/ddf.py

```diff
@@ -16,45 +16,48 @@
 #    Cyrus Harrison, Tue Feb 23 12:40:36 PST 2021
 #    Added spatial collapse with ghost zones test.
 #
+#    Kathleen Biagas, Thu Feb 17 07:38:01 PST 2022
+#    Replace old ConstructDDFAttributes attribute names for new:
+#    ddfName -> name, ranges -> binBoundaries, numSamples -> numBins,
+#    statisticalOperator -> reductionOperator,
+#    codomainName -> varForReductionOperator.
+#
```

Comment:
Are we sure we are ok possibly breaking any existing user's older `.py` files?

Do we have the option of having VisIt issue warning messages about deprecated constructs like this so they know to update their `.py` code?

Should we also have a comment in release notes about this change?

Reply:
The field name changes to ConstructDataBinningAttributes were made 12 years ago. I think we are safe.

Reply:
> Do we have the option of having VisIt issue warning messages about deprecated constructs like this so they know to update their `.py` code?

We have a ticket (#17043) that addresses this.  Title doesn't capture work to be done, see all comments.